src/transformers/generation_utils.py

```diff
@@ -660,6 +666,7 @@ def generate(
         return_dict_in_generate: Optional[bool] = None,
         forced_bos_token_id: Optional[int] = None,
         forced_eos_token_id: Optional[int] = None,
+        encoder_outputs: Optional[ModelOutput] = None,
```

Comment:
The main motivation to add this argument explicitly is so that it can be documented. Alternatively, I can keep the signature of this method untouched, and just 'silently' support `encoder_outputs`.